src/Actions/ImportProducts.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Rapidez\Statamic\Actions;
+
+use Statamic\Facades\Entry;
+use Rapidez\Statamic\Actions\CreateProducts;
+use Rapidez\Statamic\Actions\DeleteProducts;
+use Rapidez\Statamic\Events\ProductsImportedEvent;
+
+class ImportProducts
+{
+    public function __construct(
+        public CreateProducts $createProducts,
+        public DeleteProducts $deleteProducts
+    ) {
+    }
+
+    public function import(
+        ?string $store = null
+    ): void
+    {
+        $productModel = config('rapidez.models.product');
+        $storeModel = config('rapidez.models.store');
+        $storeModels = $store ? $storeModel::where('store_id', $store)->get() : $storeModel::all();
+
+        foreach($storeModels as $storeModel) {
+            config()->set('rapidez.store', $storeModel->store_id);
+
+            $products = $productModel::selectAttributes(['sku', 'name', 'url_key'])->where('catalog_product_flat_'.config('rapidez.store').'.visibility', 4)->get();
```

Comment:
How efficient is this with a lot of products? Maybe chunk the results like the Rapidez product indexer does?